packages/react-searchbox/examples/demo/src/App.js

```diff
@@ -126,11 +132,19 @@ export default () => (
             id="result-component"
             highlight
             dataField="original_title"
+            size={10}
             react={{
               and: ['search-component', 'author-filter']
             }}
           >
-            {({ results, loading }) => {
+            {({
+              results,
+              loading,
+              requestPending,
```

Comment:
@nikhil-varma Why do you have that? Please remove unused code.

Reply:
I guess this came in via the merge. I'll fix this. Let's have linters in place in our projects to inform this. Going forward this will help us a lot to mitigate such issues :D